Extensions/retags.js

```diff
@@ -181,7 +218,12 @@ XKit.extensions.retags = {
 				});
 			}
 			$t.addClass(cls);
-			$c.append($retags);
+			if (!isPopover) {
```

Comment:
Instead of propagating a isPopover function we can just return `$retags` and move the bottom code up into the tag and tagPopover functions